docs/develop/java/temporal-nexus.mdx

```diff
@@ -635,25 +641,37 @@ Use the `workflow describe` command to show pending Nexus Operations in the call
 temporal workflow describe -w <ID>
 ```
 
-Nexus events are included in the caller’s Workflow history:
+Nexus events are included in the caller's Workflow history:
 
 ```
 temporal workflow show -w <ID>
 ```
 
-For **asynchronous Nexus Operations** the following are reported in the caller’s history:
+For **asynchronous Nexus Operations** the following are reported in the caller's history:
 
 - `NexusOperationScheduled`
 - `NexusOperationStarted`
 - `NexusOperationCompleted`
 
-For **synchronous Nexus Operations** the following are reported in the caller’s history:
+For **synchronous Nexus Operations** the following are reported in the caller's history:
 
 - `NexusOperationScheduled`
 - `NexusOperationCompleted`
 
 :::note
 
-`NexusOperationStarted` isn't reported in the caller’s history for synchronous operations.
+`NexusOperationStarted` isn't reported in the caller's history for synchronous operations.
 
 :::
+
+## Learn more
+
+Evaluate if you should use Nexus and learn how it works.
+
+:::tip RELATED
```

Comment:
I'm not sure if this needs to be a tip. It looks a bit odd in a preview and this could just be bullet points.

Reply:
fixed in all places that used a tip at the end of a page for learn more.